taxonomies/nutrient_levels.txt

```diff
@@ -117,7 +117,7 @@ nb:Fett i liten kvantitet
 nd:Fat in low quantity
 ne:Fat in low quantity
 ng:Fat in low quantity
-nl:Vetten in kleine hoeveelheid
+nl:Vetten in low quantity
```

Comment:
can you remove the changes in this file? they are unrelated...

you can do "git checkout taxonomies/nutrient_levels.txt" to get back the original file